src/core/issues-notes.js

```diff
@@ -140,21 +152,30 @@ async function fetchIssuesFromGithub({ githubAPI }) {
     .map(elem => Number.parseInt(elem.dataset.number, 10))
     .filter(number => number);
   for (const issueNumber of issueNumbers) {
+    const issue = { title: "", number: issueNumber, status: "" };
     try {
       const issueURL = `${githubAPI}/issues/${issueNumber}`;
       const response = await fetch(issueURL, {
         // Get back HTML content instead of markdown
         // See: https://developer.github.com/v3/media/
         Accept: "application/vnd.github.v3.html+json",
       });
-      const issue = await response.json();
-      issues.push([issueNumber, issue]);
+      if (!response.ok) {
+        switch (response.status) {
+          case 404:
+            throw new Error("Couldn't find issue on Github. Check if it exist?");
+          default:
+            throw new Error("Network error. Github is down? or too many requests?");
+        }
+      }
+      const json = await response.json();
+      Object.assign(issue, json);
     } catch (err) {
       console.error(err);
-      const msg = `There was an error fetching ${issueNumber} from GitHub. See developer console.`;
+      const msg = `Error fetching issue "${issueNumber}" from GitHub. ${err.message}. See developer console.`;
```

Comment:
Not a big issue, but how about `#${issueNumber}` instead of `"${issueNumber}"`?

Reply:
yeah, that be nice.